Source/Widgets/InfoBox/InfoBoxViewModel.js

```diff
@@ -142,23 +161,33 @@ define([
         return (this.maxHeight - offset) + 'px';
     };
 
-    var sanitizerTaskProcessor;
-    function defaultSanitizer(html) {
-        if (!defined(sanitizerTaskProcessor)) {
-            sanitizerTaskProcessor = new TaskProcessor('sanitizeHtml', Infinity);
+    defineProperties(InfoBoxViewModel, {
+        /**
+         * Gets or sets the default HTML preprocessor function to use for all instances.
```

Comment:
I agree that this isn't my favorite way to handle this and it may be a function of not thinking about it enough.  @emackey asked for this and the\ use case is simple.  An entity has a description property that is just a key to something else (for example it's a url or id to fetch more data from a server).  Since Entity properties aren't asynchronous, the easiest way to handle this with built in functionality is to use this preprocessor (which is a promise) to turn the description into what you really want.

Now that I'm thinking about it, there's got to be a cleaner way to do that, especially given that we have an iframe now.  I can't think of another use case other than this one that would warrant us keeping this around.